src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/LockFileUtils.cs

```diff
@@ -138,10 +152,19 @@ internal static List<List<SelectionCriteria>> CreateOrderedCriteriaSets(RestoreT
             return orderedCriteriaSets;
         }
 
+        private static void ApplyAliases(LibraryDependency libraryDependency, LockFileItem item)
+        {
+            if (libraryDependency?.Aliases != null)
```

Comment:
Will the empty aliases need to be applied? I think that the non-sdk style projects will have empty Aliases if the the Aliases metadata is not defined.

Reply:
They shouldn't be. I think you are about those being potentially null. 

I'll take a look.

Reply:
You were right, I fixed that. Nice catch.